### ISSUE TITLE
Set Mail_URL for Direct Contact (On Hold)

### ISSUE THREAD

**Issue by luisy-eng** [manoa-inspire/MATP#48]
The email service is mostly complete on the Contact Us page; we need to integrate and test the actual email service. 

Set the MAIL_URL Environment Variable
Choose an SMTP server:

Use any SMTP provider, such as Gmail.
For Gmail, it requires enabling less secure apps.
Set the MAIL_URL variable: In the server/main.js file, set the MAIL_URL environment variable. Here is an example for Gmail:

// Set up SMTP settings using Gmail
Meteor.startup(() => {
  process.env.MAIL_URL = 'smtp://your-email@gmail.com:your-password@smtp.gmail.com:465/';
});

Explanation:
your-email@gmail.com: Replace this with your Gmail address.
your-password: Replace this with your Gmail password.
smtp.gmail.com:465: This is the Gmail SMTP server.

Test the Email:
Once the MAIL_URL is correctly set up, try submitting the form. The email should be sent successfully.

Once this is complete, we can try to make it work for the company, but it would be best to ask the professor for help after we have tested it.
